src/main/java/org/jitsi/jicofo/ParticipantChannelAllocator.java

```diff
@@ -257,11 +257,11 @@ private boolean doInviteOrReinvite(
             JicofoJingleUtils.addStartMutedExtension(
                 jingleIQ, startMuted[0], startMuted[1]);
         }
-        String serverRegion = bridgeSession.bridge.getRegion();
-        if (serverRegion != null)
-        {
-            jingleIQ.addExtension(new ServerRegionPacketExtension(serverRegion));
-        }
+
+        // Include info about the BridgeSession which provides the transport
+        jingleIQ.addExtension(
+            new BridgeSessionPacketExtension(
```

Comment:
Have clients adapted to the new format?

Reply:
Yes, there's another PR to lib-jitsi-meet https://github.com/jitsi/lib-jitsi-meet/pull/862